cpp/src/Slice/Grammar.y

```diff
@@ -1737,7 +1737,7 @@ interface_list
     unit->error("illegal inheritance from type Object");
     $$ = new InterfaceListTok; // Dummy
 }
-| ICE_VALUE
+| ICE_ANYCLASS
 {
     unit->error("illegal inheritance from type Value");
```

Comment:
The error message should use `AnyClass`.

Reply:
Fixed!

Reply:
Actually... this error message means I can't truly alias `Value` and `AnyClass` and have to keep tokens for both! Because if they actually use the old `Value` keyword, we also need an error message to handle it too...

Reply:
Another advantage of keeping both is a more logical behavior in compat-mode:

An `Object` parameter is converted to `AnyClass?`.
A `Value` parameter is likewise converted to `AnyClass?`.
An `AnyClass` parameter remains an `AnyClass` parameter.